tests/data/example_extension.model.csv

```diff
@@ -0,0 +1,21 @@
+Attribute,Description,Valid Values,DependsOn,Properties,Required,Parent,DependsOn Component,Source,Validation Rules
```

Comment:
Could you provide more context on why this additional CSV file is needed? I hesitate to have so much duplicated content that we would have to keep in sync. Would we be able to load the existing CSV data model and update it with code?

Reply:
Right, I created this for a couple of unit tests that check if the package is able to create a new extended JSON-LD schema (from `example_extension.model.csv`) from a given base schema JSON-LD (from `example.model.csv`).

Reply:
If possible, I would like to capture those expectations using code instead of a mostly-duplicated CSV data model. For example, if you're adding an attribute, you can look for that attribute in the extended model and confirm some of its keys and values.